src/content/blog/astro-560.mdx

```diff
@@ -0,0 +1,170 @@
+import { API_URL } from 'astro:env/server';
+import { createClient } from './client.js';
+
+// Astro 5.5: undefined
+// Astro 5.6: string
+const client = createClient(API_URL);
+
+export const onRequest = defineMiddleware((ctx, next) => {
+	ctx.locals.client = client;
+	return next();
+});
+```
+
+This improvement creates a more consistent developer experience across all Astro server environments.
+
+## Experimental sessions on Cloudflare
+
+The [Astro sessions API](https://docs.astro.build/en/reference/experimental-flags/sessions/) is an experimental feature that allows you to easily store user data between requests. It uses pluggable backends for storage, which are automatically configured when using the Node or Netlify adapters.
+
+With Astro 5.6, we've brought that simplified integration to Cloudflare, too. Astro now automatically configures Cloudflare KV storage when you're using sessions with the Cloudflare adapter. This means your session data can be reliably stored and accessed across Cloudflare's global network with minimal configuration.
+
+Getting started is three steps:
+
+1. Create a KV namespace using the Wrangler CLI:
+
+```sh
+npx wrangler kv namespace create "SESSION"
+```
+
+2. Declare the KV namespace in your Wrangler config:
+
+```jsonc
+// wrangler.json
+{
+	"kv_namespaces": [
+		{
+			"binding": "SESSION",
+			"id": "<SESSION_ID>"
+		}
+	]
+}
+```
+
+3. Enable experimental sessions in your Astro config:
+
+```js
+// astro.config.mjs
+export default defineConfig({
+	adapter: cloudflare(),
+	experimental: {
+		sessions: true,
+	},
+});
+```
+
+You can then use sessions in your server code:
+
+```astro
+---
+export const prerender = false;
+const cart = await Astro.session.get('cart');
+---
+
+<a href="/checkout">🛒 {cart?.length ?? 0} items</a>
+```
+
+For more information, see the [experimental sessions docs](https://docs.astro.build/en/reference/experimental-flags/sessions/).
+
+## New Prefetch Eagerness Option
+
+Find the perfect balance between speed and resource usage with Astro's new prefetch eagerness controls. With the experimental [`clientPrerender`](https://docs.astro.build/en/reference/experimental-flags/client-prerender/) flag enabled, you can use the `eagerness` option on `prefetch()` to suggest to the browser how eagerly it should prefetch/prerender link targets.
+
+This new option, which aligns with the browser's [Speculation Rules API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/speculationrules), gives you fine-grained control over how aggressively the browser should prefetch or prerender your links:
+
+```astro
+---
+---
+<script>
+// Control prefetching eagerness
+import { prefetch } from 'astro:prefetch';
+
+// Let's be strategic about this resource-intensive page
+prefetch('/data-heavy-dashboard', { eagerness: 'conservative' });
+
+// This page is critical to the user journey, load it ASAP!
+prefetch('/product-details', { eagerness: 'immediate' });
```

Comment:
```suggestion
// This page is critical to the user journey, load it ASAP!
prefetch('/product-details'); // defaults to `{ eagerness: 'immediate' }`
```
Wondering about showing it like this, maybe, since it's the default and no one needs to write this out in full?